Dockerfile

```diff
@@ -17,10 +17,14 @@ RUN openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /certs/cert.key
 #
 FROM --platform=${TARGETPLATFORM:-linux/amd64} nginx:alpine AS base
 COPY --from=certificate /certs/cert.pem /certs/cert.pem
+COPY --from=certificate /certs/cert.pem /certs/CA.pem
 COPY --from=certificate /certs/cert.key /certs/cert.key
 RUN chmod g+rwx /etc/nginx/nginx.conf /etc/nginx/conf.d /etc/nginx/conf.d/default.conf /var/cache/nginx /var/run /var/log/nginx /etc/ssl /certs
 ENV SSL_CERTIFICATE=/certs/cert.pem
 ENV SSL_CERTIFICATE_KEY=/certs/cert.key
+ENV SSL_CLIENT_CERTIFICATE=/certs/CA.pem
+ENV EDA_WEBHOOK_SERVER=${EDA_WEBHOOK_SERVER:-http://example.com}
```

Comment:
These are EDA specific and should be in the eda-ui image only, not the base image